spec/models/listing_spec.rb

```diff
@@ -2,27 +2,70 @@
 
 RSpec.describe Listing do
   before(:each) do
-    @listing = create(:listing)
+    # @listing = create(:listing)
+
+    @listing_1 = create(:listing, year: 2001, make: "Toyota", model: "Camry")
+    @listing_2 = create(:listing, year: 2002, make: "Toyota", model: "4Runner")
+    @listing_3 = create(:listing, year: 2003, make: "Toyota", model: "Tundra")
+    @listing_4 = create(:listing, year: 2004, make: "Chevrolet", model: "Suburban")
+    @listing_5 = create(:listing, year: 2005, make: "Chevrolet", model: "Silverado")
+    @listing_6 = create(:listing, year: 2006, make: "Ford", model: "F-150")
+    @listing_7 = create(:listing, year: 2007, make: "Ford", model: "F-150")
+    @listing_8 = create(:listing, year: 2008, make: "Ford", model: "Focus")
+    @listing_9 = create(:listing, year: 2009, make: "Ford", model: "Escape")
+    @listing_10 = create(:listing, year: 2010, make: "Ford", model: "Escape")
+
   end
 
   it 'exists' do
-    expect(@listing).to be_a(Listing)
+    expect(@listing_1).to be_a(Listing)
   end
 
   it 'has attributes' do
```

Comment:
Maybe segregate these tests with a describe or a context block in the future to help break up the tests. Otherwise these are great.